benchmarking/nursery/benchmark_automl/launch_remote.py

```diff
@@ -14,12 +14,14 @@
 from pathlib import Path
```

Comment:
David asked me to leave this code unchanged (everything in `nursery/benchmark_automl`)

Reply:
Ok I se the point of not updating the versions to make sure the results from the paper are reproducible. However, we are updaing the syne-tune version (since code is pulled from local source rather than fixed version) so I think we should update the container used for execution or otherwise it might not be able to install up-to-date dependencies.